runtime/battery-station/src/xcm_config/asset_registry.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Zeitgeist PM LLC.
+// Copyright 2022-2023 Forecasting Technologies LTD.
```

Comment:
Is it because `2022` the change of entities happened? Is it sufficient to only name one in that case?

Reply:
> Is it because `2022` the change of entities happened?

Yes. Forecasting Technologies took over in October 2022, and the `asset_registry.rs` was created in November of the same year: https://github.com/zeitgeistpm/zeitgeist/commits/0e5af7f33a0445c72e72e0a1a1378c05157dd565/runtime/battery-station/src/xcm_config/asset_registry.rs.

> Is it sufficient to only name one in that case?

Seem reasonable to me. Everyone working on this file was employed by Forecasting Technologies. Why would Zeitgeist PM retain a copyright? That being said, I'm making all of this up on the spot using only common sense. If you want to know for sure (including how copyright notices and licenses should be managed), you should ask a lawyer.